miner/worker.go

```diff
@@ -287,6 +294,31 @@ func (miner *Miner) commitTransaction(env *environment, tx *types.Transaction) e
 	if tx.Type() == types.BlobTxType {
 		return miner.commitBlobTransaction(env, tx)
 	}
+
+	// If a conditional is set, check prior to applying
+	if conditional := tx.Conditional(); conditional != nil {
+		now, cost := time.Now(), conditional.Cost()
+		res := miner.conditionalLimiter.ReserveN(now, cost)
+		if !res.OK() {
+			res.Cancel()
```

Comment:
Pedantic but this isn't necessary.

https://pkg.go.dev/golang.org/x/time/rate#Reservation.OK

https://github.com/ethereum-optimism/op-geth/compare/sendRawTransactionConditional...anacrolix/sendRawTransactionConditional#diff-689874b30f6f905ce6c47cdefeddcf052b467a7936a981f4b6cf38939e10d924R302-R311

https://cs.opensource.google/go/x/time/+/refs/tags/v0.6.0:rate/rate.go;l=169-171

Reply:
ah nice